README.md

```diff
@@ -6,12 +6,24 @@ This is a GitHub action for using setting up `captain` in your job.
 
 ### `version`
 
-The version of `captain` to install. Defaults to `latest`.
+The version of `captain` to install. Defaults to `v1`. Possible values are `v1`,
+`latest`, or a semantic version number.
+
+`latest` will look up the latest release across all major versions.
```

Comment:
Let's remove mention of `latest` from the README. I don't think we want anybody to use it since it's nearly guaranteed to break their builds all of a sudden when we ship a new major version.